.run/generateConfigs.py

```diff
@@ -41,7 +41,7 @@ class IdeVariant:
 </component>'''
 
 if __name__ == '__main__':
-    mvs = ["2023.2", "2023.3", "2024.1"]
+    mvs = ["2023.2", "2023.3", "2024.1", "2024.2"]
```

Comment:
Can we also add this to the prelease GH workflow?

Reply:
let's do separately